bindings/python/linkgrammar.py

```diff
@@ -51,10 +50,6 @@ def __del__(self):
             del self._obj
 
     @property
-    def version(self):
-        return clg.linkgrammar_get_version()
```

Comment:
Of course.
The function linkgrammar_get_version() is fine and a good idea.
It is just not a Parse Option.
It is still accessible as clg.linkgrammar_get_version().
Did I miss something?